### DISCUSSION TITLE
Local Adjustments Presets, am I missing something?

### DISCUSSION THREAD

**Discussion by dtucker-art** [rsjaffe/MIDI2LR#601] [Problems]
I just installed everything over onto my new Mac Mini M1 and all is well except that the **Local adjustment presets** are not showing up in the pop up list in the plug in extras in general options and then local adjustments presets. Is this a glitch, or am I doing something wrong? Thanks in advance for any help you can offer. It's been a little while since I installed everything on my old computer so I don't remember everything perfectly. 

I should add that when I moved over to the new computer and upgraded LR, the old local adjustment presets were gone. I couldn't figure out how to reinstall them, so I just recreated them. So they are there in the "custom" list. Thanks!

**Comment by rsjaffe** [comment]
Restart Lightroom. New presets won't show up in MIDI2LR until after a restart.

**Comment by dtucker-art** [comment]
Thanks so much for your reply. I did try restarting LRc as well as 
computer itself, no luck however. Still nothing in the pop up lists.

**Comment by dtucker-art** [comment]
Is there not anyone who can help with this issue? Are there other resources I could go to try and solve this problem? TIA

**Reply by rsjaffe** [reply]
MIDI2LR looks for the local adjustment presets in the folder and subfolders of the following, depending upon whether you have a Mac or Windows machine. It will not find local adjustment presets that are stored elsewhere, which is what I think happened when you added those in.

/Users/[user name]/Library/Application Support/Adobe/Lightroom/Local Adjustment Presets
C:\Users\[user name]\AppData\Roaming\Adobe\Lightroom\Local Adjustment Presets

**Comment by dtucker-art** [comment]
Thank You so much that did the trick!!

I had searched for this folder, but for some crazy reason Mac seems to 
ignore everything in the Library folder.

Thanks again!!

David

**Comment by ERNESTDL** [comment]
I just installed MIDI2LR on a MIDI KORG NanoKontrol 2

I use lightroom classic 11.3.1

I would like to know how to assign to MIDI the presets that I normally use, I can't find the way to do it, does anyone have a link or a tutorial to do it?

Thanks
